test/test.js

```diff
@@ -104,3 +105,8 @@ test("allow to pass not string values to worker", async () => {
     '"use strict";window.a=5,window.a<3&&console.log(4);\n'
   );
 });
+
+test("normalizeOptions removes 'sourcemap'", async () => {
+  const normalizedOptions = normalizeOptions({ sourcemap: true })
+  expect(normalizedOptions).toEqual({ sourceMap: true });
```

Comment:
I don't need unit test. The whole plugin should work.

Reply:
I'll remove that commit then

Reply:
it's gone now